lib/dashlib.py

```diff
@@ -11,6 +11,8 @@
 from misc import printdbg, epoch2str
 import time
 
+MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024
```

Comment:
Is it possible that `MAX_GOVERNANCE_OBJECT_DATA_SIZE` can/will be changed in the future? If so, then we should add it to the output of `getgovernanceinfo` and use that value from dashd instead of hard-coding it here and having to change the code in 2 places (Core + Sentinel).

Reply:
Good point. I think it can be changed, so I'll update the PR.